src/app/forms/templates/hgv/hgv-tech-record.template.ts

```diff
@@ -141,10 +141,10 @@ export const HgvTechRecord: FormNode = {
           value: '',
           type: FormNodeTypes.CONTROL,
           editType: FormNodeEditTypes.SELECT,
-          options: getOptionsFromEnum(VehicleClass.DescriptionEnum)
+          options: getOptionsFromEnum(VehicleClass.DescriptionEnum),
+          validators: [{ name: ValidatorNames.Required }]
         }
-      ],
-      validators: [{ name: ValidatorNames.Required }]
```

Comment:
Is there an impact on amending vehicle by removing this validators attribute seen as though create and amend share the same templates

Reply:
That involves extracting some logic from this component and the one you mentioned into a generic class which then these will inherit. That is a great suggestion, but not MVP at this moment and is beyond the scope of this ticket.

Should mention it in the sprint retro though! We've had quite a few refactoring PRs lately.

Reply:
You want a required on the empty object instead of a value? I just moved it up two lines

Reply:
I'll not be at the retro, I'll leave that action with you ;)

Reply:
I was just wondering how the two might differ

Reply:
I can stick it on both if you like :o it just seemed really redundant